.github/CONTRIBUTING.md

```diff
@@ -371,13 +371,45 @@ This does not go for bug reports however, as we want to unearth and solve as man
 
 ### Commit
 
-* Focus on one thing per commit 
-* Keep changes as small as possible 
-* Only commit related functinoality for each commit 
+You may have encountered statements like
+
+> Commit early, Commit often
+
+and
+
+> Commit Often, Perfect Later, Publish Once
+>
+> _[Git Best Practices](https://sethrobertson.github.io/GitBestPractices/)_
+
+#### The good commit follows Git best practices
+
+You are encouraged to follow commonly adopted Git best practices, see for example [Distributed Git - Contributing to a Project](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project). These include, among others:
```

Comment:
Should we change the wording, to more something along the lines of "we would like to highlight[...]"? This should be to point out that we perhaps care a bit more about these, than the rest in the _Distributed Git_-article. 

```suggestion
You are encouraged to follow commonly adopted Git best practices, see for example [Distributed Git - Contributing to a Project](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project). While all of the practices in that article is useful, we would especially like to highlight:
```